### PR TITLE
Add dotnet SDK manifest

### PR DESCRIPTION
**Pull request by sacOO7**
- Fixed https://github.com/ably/ably-dotnet/issues/1175

### CODE REVIEW

**Review by QuintinWillison — COMMENTED** [review]
I wonder if this may not be complete as when I view the [rendered matrix](https://sdk.ably.com/builds/ably/features/pull/96/matrix/) then I notice we're indicating lack of compliance with `Protocol: MessagePack`, which I was under the belief was implemented in this SDK. Is that not the case?

Unrelated, as far as I can tell the lack of `Agent Identifier` is because [#498](https://github.com/ably/ably-dotnet/issues/498) was closed without being implemented ([#607](https://github.com/ably/ably-dotnet/pull/607) was created but then closed without explanation).

**Comment by sacOO7** [comment]
> I wonder if this may not be complete as when I view the [rendered matrix](https://sdk.ably.com/builds/ably/features/pull/96/matrix/) then I notice we're indicating lack of compliance with `Protocol: MessagePack`, which I was under the belief was implemented in this SDK. Is that not the case?
> 
> Unrelated, as far as I can tell the lack of `Agent Identifier` is because [#498](https://github.com/ably/ably-dotnet/issues/498) was closed without being implemented ([#607](https://github.com/ably/ably-dotnet/pull/607) was created but then closed without explanation).

I was working on the message pack, you can see the PR https://github.com/ably/ably-dotnet/pull/494, but I was asked to stop the work in the interest of other high-priority work, then we never picked it up from there. 
But I can say, we had made some progress with it. 
There seems to be a new issue created to resolve this -> https://github.com/ably/ably-dotnet/issues/590.

**Review by QuintinWillison — APPROVED** [review]

**Comment by sacOO7** [comment]
Seems I don't have permission to merge this PR : (

File: sdk-manifests/ably-dotnet.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        Member List:
+      Publish:
+        Idempotence:
+      Push Notifications:
+        List Subscriptions:
+        Subscribe:
+      Release:
+      Status:
+        Channel Details:
+        Enumerate:
```
Comment (owenpearson) [inline sdk-manifests/ably-dotnet.yaml:76]: I don't think this feature is implemented in ably-dotnet (or any SDK). Just to be clear this one is about using the channel enumeration API - not about enumerating local RestChannel instances.

**Review by owenpearson — CHANGES_REQUESTED** [review]
A couple of notes:
- `Transport: Connection Open Timeout` seems to be implemented in ably-dotnet
- It seems like there's a msgpack serialiser and tests in the ably-dotnet repo, is there no way for developers using the library to enable it? If there is then we should probably add it with a caveat.

**Comment by sacOO7** [comment]
> * Timeout

`Transport: Connection Open Timeout` is just provided as a option under `ClientOptions`. If you try to find usages, you will find none.

**Comment by sacOO7** [comment]
> A couple of notes:
> 
> * `Transport: Connection Open Timeout` seems to be implemented in ably-dotnet
> * It seems like there's a msgpack serialiser and tests in the ably-dotnet repo, is there no way for developers using the library to enable it? If there is then we should probably add it with a caveat.

`Transport: Connection Open Timeout` is just provided as a option under `ClientOptions`. If you try to find usages, you will find none.

File: sdk-manifests/ably-dotnet.yaml (hunk 1 above)
Comment (sacOO7) [reply sdk-manifests/ably-dotnet.yaml:76]: Actually, I had a doubt about this one, I will provide link to the `enumerator` 
https://github.com/ably/ably-dotnet/blob/b6088aa985068535ce6f30f072399aea612fc478/src/IO.Ably.Shared/Realtime/RealtimeChannels.cs#L193-L206

Comment (sacOO7) [reply sdk-manifests/ably-dotnet.yaml:76]: Okay, seems it's a part of `Status->Enumerate` API, I will update

Comment (sacOO7) [reply sdk-manifests/ably-dotnet.yaml]: Removed

**Comment by owenpearson** [comment]
@sacOO7 `HttpOpenTimeout` Is used [here](https://github.com/ably/ably-dotnet/blob/b6088aa985068535ce6f30f072399aea612fc478/src/IO.Ably.Shared/AblyRest.cs#L433)

**Comment by sacOO7** [comment]
> @sacOO7 `HttpOpenTimeout` Is used [here](https://github.com/ably/ably-dotnet/blob/b6088aa985068535ce6f30f072399aea612fc478/src/IO.Ably.Shared/AblyRest.cs#L433)

It's using the value from `Defaults.MaxHttpOpenTimeout` and not from `clientOptions.httpOpenTimeout`

**Comment by owenpearson** [comment]
@sacOO7 I stand corrected :)

**Review by owenpearson — APPROVED** [review]

**Pull request merged**